exporter/opensearchexporter/README.md

```diff
@@ -0,0 +1,66 @@
+# OpenSearch Exporter
+
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [devel]   |
+| Supported pipeline types | traces    |
+| Distributions            | [contrib] |
+
+This exporter supports sending OpenTelemetry signals as documents to [OpenSearch](https://www.opensearch.org).
+
+The documents are sent using [observability catalog](https://github.com/opensearch-project/opensearch-catalog/tree/main/schema/observability) schema.
+
+## Configuration options
+
+- `endpoints`: List of OpenSearch URLs. If endpoints is missing, the
+  OPENSEARCH_URL environment variable will be used.
+- `num_workers` (optional): Number of workers publishing bulk requests concurrently.
+- `dataset` (default=`default`) a user-provided label. It is used to construct the name of the destination index or data stream.
+- `namespace` (default=`namespace`)a user-provided label. It is used to construct the name of the destination index or data stream.
```

Comment:
Is `namespace` a sensible default value here?  It feels odd but I'm not super familiar with typical OpenSearch usage.  If namespaces are optional then a default of an empty string might make more sense, but I'll have to defer to your expertise here.

Reply:
This is following the specification [here](https://github.com/opensearch-project/opensearch-catalog/blob/main/docs/schema/observability/Naming-convention.md).

@YANG-DB  Does default namespace value of `namespace` crucial?

Reply:
Hi
yes - without any other specific index name we fallback to this naming convention 
we have an index pattern for all signal types to catch these index names:
 - `ss4o_logs-*-*`
 - `ss4o_traces-*-*`
 - `ss4o_metrics-*-*`